src/main/java/org/sonarlint/intellij/ui/ruledescription/RuleHeaderPanel.kt

```diff
@@ -43,6 +46,10 @@ import javax.swing.SwingConstants
 
 
 class RuleHeaderPanel : JBPanel<RuleHeaderPanel>(FlowLayout(FlowLayout.LEFT)) {
+    companion object{
+        val CHANGE_STATUS = "Change Status"
```

Comment:
Maybe we should keep "Change Status" for security hotspots and have "Mark as Resolved" for issues?